yarn-project/end-to-end/src/e2e_fees.test.ts

```diff
@@ -275,6 +278,114 @@ describe('e2e_fees', () => {
     );
   });
 
+  it('fails transaction that error in setup', async () => {
+    const OutrageousPublicAmountAliceDoesNotHave = 10000n;
+    // const PublicMintedAlicePublicBananas = 1000n;
+    const FeeAmount = 1n;
+    const RefundAmount = 2n;
+    const MaxFee = FeeAmount + RefundAmount;
+    const { wallets } = e2eContext;
+
+    // simulation throws an error when setup fails
+    await expect(
+      bananaCoin.methods
+        .transfer_public(aliceAddress, sequencerAddress, OutrageousPublicAmountAliceDoesNotHave, 0)
+        .send({
+          fee: {
+            maxFee: MaxFee,
+            paymentMethod: new BuggedSetupFeePaymentMethod(bananaCoin.address, bananaFPC.address, wallets[0]),
+          },
+        })
+        .wait(),
+    ).rejects.toThrow(/Message not authorized by account 'is_valid == true'/);
+
+    // so does the sequencer
+    await expect(
+      bananaCoin.methods
+        .transfer_public(aliceAddress, sequencerAddress, OutrageousPublicAmountAliceDoesNotHave, 0)
+        .send({
+          skipPublicSimulation: true,
+          fee: {
+            maxFee: MaxFee,
+            paymentMethod: new BuggedSetupFeePaymentMethod(bananaCoin.address, bananaFPC.address, wallets[0]),
+          },
+        })
+        .wait(),
+    ).rejects.toThrow(/Transaction [0-9a-f]{64} was dropped\. Reason: Tx dropped by P2P node\./);
+  });
+
+  it('fails transaction that error in teardown', async () => {
+    /**
```

Comment:
Is this comment correct? Aren't we using a bad nonce instead?

Reply:
It is correct: we're using a bad nonce in teardown! In `BuggedTeardownFeePaymentMethod` at the bottom of this test, it specifies a transfer as the teardown function, and it is in that transfer that we use the bad nonce.